modules/nw-route-admission-policy.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+// * networking/routes/route-configuration.adoc
+
+[id="nw-route-admission-policy_{context}"]
+= Configuring the route admission policy
+
+Cluster administrators can run applications in multiple namespaces with the same domain name by configuring the `RouteAdmissionPolicy` `ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK` environment variable.
+
+.Prerequisites
+
+* Cluster administrator privileges.
+
+.Procedure
+
+* Edit the Ingress Controller to configure the `RouteAdmissionPolicy` `ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK` environment variable using the following command:
```

Comment:
We'll be editing the `.spec.routeAdmission` field of the `ingresscontroller` resource. To make the example more concrete, it could be framed as an edit to the well-known default ingresscontroller, like:

`oc edit --namepace openshift-ingress-operator ingresscontrollers/default`

and setting `.spec.routeAdmission.namespaceOwnership=InterNamespaceAllowed`

We don't want any references to ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK anywhere.